app/src/androidTest/java/com/github/sdp/ratemyepfl/database/reviewable/ReviewableRepositoryImplTest.kt

```diff
@@ -0,0 +1,231 @@
+package com.github.sdp.ratemyepfl.database.reviewable
+
+import com.github.sdp.ratemyepfl.database.SearchableRepository
+import com.github.sdp.ratemyepfl.database.query.QueryResult.Companion.mapEach
+import com.github.sdp.ratemyepfl.database.query.QueryState
+import com.github.sdp.ratemyepfl.database.reviewable.CourseRepositoryImpl.Companion.toCourse
+import com.github.sdp.ratemyepfl.database.util.RepositoryUtil
+import com.github.sdp.ratemyepfl.model.items.Course
+import com.google.firebase.firestore.FirebaseFirestore
+import com.google.firebase.firestore.Query
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
+import junit.framework.Assert.assertEquals
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.tasks.await
+import kotlinx.coroutines.test.runTest
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import java.lang.Integer.min
+import javax.inject.Inject
+
+private typealias ReviewableItem = Course
+
+@ExperimentalCoroutinesApi
+@HiltAndroidTest
+class ReviewableRepositoryImplTest {
+
+    @Inject
+    lateinit var db: FirebaseFirestore
+
+    private lateinit var repository: ReviewableRepositoryImpl<ReviewableItem>
+
+    @get:Rule
+    val hiltRule = HiltAndroidRule(this)
+    private val fake = "fake"
+    private val fakeTeacher = fake
+    private val personalizedTeacher = "myPersonalTeacher"
+    private val courseBuilder = Course.Builder(
+        fake,
```

Comment:
I think it is better to put the arguments on one line

Reply:
Android studio made this with the formatting. I think we don't have the choice :(

Reply:
Actually you can put manually them in one line and Android Studio will not say anything

Reply:
Wow, it does indeed. I was not aware of this x) I'll change it.